prog/mutation.go

```diff
@@ -235,29 +235,9 @@ func (t *IntType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []*Ca
 
 func (t *FlagsType) mutate(r *randGen, s *state, arg Arg, ctx ArgCtx) (calls []*Call, retry, preserve bool) {
 	a := arg.(*ConstArg)
-
 	for oldVal := a.Val; oldVal == a.Val; {
-		// Generate a new value.
-		if r.nOutOf(1, 5) {
-			a.Val = r.flags(t.Vals, t.BitMask, 0)
-			continue
-		}
-
-		if !t.BitMask || (t.BitMask && r.nOutOf(1, 4)) {
-			a.Val = r.flags(t.Vals, t.BitMask, a.Val)
-			continue
-		}
-
-		for stop := false; !stop; stop = r.oneOf(3) {
-			idx := r.rand(len(t.Vals))
-			if r.bin() {
-				a.Val |= 1 << t.Vals[idx]
-			} else {
-				a.Val &= ^(1 << t.Vals[idx])
-			}
-		}
+		a.Val = r.flags(t.Vals, t.Enum, t.BitMask, a.Val)
```

Comment:
Having more unification between generation and mutation is good, but this undoes   @VeronicaRadu change to (1) generate flags from scratch, (2) reset bits in the existing value.

Reply:
I see that #1362 fixes this